README.md

```diff
@@ -156,7 +156,7 @@ Here is a list of third-party **tools** maintained by the community:
 - [Kong-UI](https://github.com/msaraf/kong-ui): Admin UI in JavaScript
 - [Konga](https://github.com/Floby/konga-cli): CLI Admin tool in JavaScript
 - [Konga](https://pantsel.github.io/konga/): Another elegant Kong UI in Javascript
-- [Kongfig](https://github.com/mybuilder/kongfig): Declarative configuration in JavaScript
+- [Kongfig](https://github.com/mybuilder/kongfig): Declarative Kong configuration in YAML, JSON or JavaScript
```

Comment:
Should this say "*in* JavaScript"? I was under the impression that JavaScript was just the implementation language for Kongfig; unless there is now also a JavaScript "SDK" for Kong configuration?

Reply:
Not sure I understand this comment. "_in_ YAML, JSON, or JavaScript" implies "_in_ JavaScript". There is some ambiguity here, but the meaning leans more toward the meaning derived from _in_ than an inherent implication of a JavasScript implementation.

Reply:
It does support writing the config in JS as well, useful when having slightly different env and you need all the JS scripting power :)

Reply:
Maybe it would be better to add links to the sample config files; Declarative Kong configuration in [YAML](config), [JSON](https://github.com/mybuilder/kongfig/blob/master/config.json.sample), or [JavaScript](https://github.com/mybuilder/kongfig/blob/master/config.js.sample)

Reply:
That helps! Thank you for the clarification. I don't think we should include additional links (no other community-contributed module has more than a single link to it in this list), but the oxford comma that you added in your last proposal sounds good :) 

Thank you!